conf/base.py

```diff
@@ -83,11 +83,22 @@
     SECURE_CONTENT_TYPE_NOSNIFF = True
     SECURE_BROWSER_XSS_FILTER = True
     X_FRAME_OPTIONS = 'DENY'
+    SCHEMA = 'https://'
+    S3_HOST = 's3.amazonaws.com'
 else:
     DEBUG = True
     LOG_LEVEL = 'DEBUG'
     DEV_SECRET_KEY = 'devsecretkey' * 19  # noqa
     SECRET_KEY = os.environ.get('SECRET_KEY', DEV_SECRET_KEY)
+    SCHEMA = 'http://'
+    S3_HOST = 'minio:9000'
+
+# Aws s3 credentials
+S3_BUCKET = f"document-management-s3-{ENVIRONMENT.lower()}"
+AWS_SERVER_KEY = os.environ.get('AWS_SERVER_KEY', 'TEST-DEV-KEY')
```

Comment:
Don't include settings for AWS keys. boto3 will pick these up automatically if they are set in the environment (make sure you set them in dev.yml) https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#environment-variables